src/DIRAC/ProductionSystem/scripts/dirac_prod_delete.py

```diff
@@ -1,7 +1,15 @@
 #!/usr/bin/env python
 
 """
-  Delete a given production
+Delete a given production
+
+Usage::
```

Comment:
Single colons (`Usage:`) should be enough, right? is in lots of places, also Arguments, Examples.

Reply:
yes

Reply:
corrected everywhere to the format:
```
"""
Script description

Usage:
  script [options] ...

Example:
  $ script
"""
```